testing/benchmark_hooks.py

```
@@ -0,0 +1,95 @@
+    glob.glob("testing/**/*.c", recursive=True)
+    + glob.glob("testing/**/*.cpp", recursive=True)
+    + glob.glob("testing/**/*.h", recursive=True)
+    + glob.glob("testing/**/*.hpp", recursive=True)
+)
+
+REPEATS = 5
+RESULTS_FILE = "testing/benchmark_results.txt"
+
+
+def run_hook(config, files):
+    cmd = ["pre-commit", "run", "--config", config, "--files"] + files
```

Ruff (0.12.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)